L1Trigger/Phase2L1ParticleFlow/src/PFAlgo3.cc

```diff
@@ -55,10 +55,11 @@ PFAlgo3::PFAlgo3( const edm::ParameterSet & iConfig ) :
     tightTrackMinStubs_ = linkcfg.getParameter<unsigned>("tightTrackMinStubs");
     tightTrackMaxChi2_  = linkcfg.getParameter<double>("tightTrackMaxChi2");
     tightTrackMaxInvisiblePt_ = linkcfg.getParameter<double>("tightTrackMaxInvisiblePt");
+    sortInputs_ = iConfig.getUntrackedParameter<bool>("sortInputs");
```

Comment:
you forgot this one

Reply:
whoops, I changed that line but forgot also switch that, thanks for catching it